tests/Models/Article.php

```diff
@@ -14,4 +14,9 @@ public function User()
     {
         return $this->belongsTo(User::class);
     }
+
+    public function SnakeUser()
```

Comment:
Shouldn't it be `snakeUser()`? 🤔
And could you add this relationship only in the anonymous class in the test method?

Reply:
Yes and yes. Sorry for my sloppiness 😊